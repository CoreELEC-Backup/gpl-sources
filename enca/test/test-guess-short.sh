#! /bin/sh
# Purpose: test charset recognition on tiny text fragments.
. $srcdir/setup.sh
for l in $ALL_TEST_LANGUAGES; do
  $ENCA -p -e -L $l $srcdir/$l-s.* | sed -e "s#^$srcdir/##" >>$TESTNAME.actual || DIE=1
done
. $srcdir/finish.sh
