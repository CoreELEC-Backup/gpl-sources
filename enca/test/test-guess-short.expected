be-s.cp1251: CP1251
be-s.ibm855: IBM855
be-s.ibm866: IBM866
be-s.iso88595: ISO-8859-5
be-s.koi8uni: KOI8-UNI
be-s.maccyr: maccyr
be-s.utf8: UTF-8/CRLF
bg-s.cp1251: CP1251
bg-s.cp1251.qp: CP1251/LF/qp
bg-s.ecma113: ECMA-113
bg-s.ibm855: IBM855
bg-s.iso88595: ISO-8859-5
bg-s.maccyr: maccyr
bg-s.utf8: UTF-8/CRLF
cs-s.ambi: CP1250
cs-s.cork: CORK/LF
cs-s.cp1250: CP1250
cs-s.ibm852: IBM852
cs-s.ibm852.qp: IBM852/qp
cs-s.iso88592: ISO-8859-2
cs-s.keybcs2: KEYBCS2
cs-s.koi8cs2: KOI-8_CS_2
cs-s.macce: macce
cs-s.tex: TeX
cs-s.utf8: UTF-8/CRLF
et-s.baltic: baltic
et-s.cp1257: CP1257
et-s.ibm775: IBM775
et-s.iso885913: ISO-8859-13
et-s.iso88594: ISO-8859-4
et-s.iso88594.qp: ISO-8859-4/qp
et-s.macce: macce
et-s.utf8: UTF-8/CRLF
hr-s.cork: CORK/LF
hr-s.cp1250: CP1250
hr-s.ibm852: IBM852
hr-s.iso88592: ISO-8859-2
hr-s.iso88592.qp: ISO-8859-2/qp
hr-s.macce: macce
hr-s.utf8: UTF-8/CRLF
hu-s.cork: CORK/LF
hu-s.cp1250: CP1250
hu-s.ibm852: IBM852
hu-s.iso88592: ISO-8859-2
hu-s.macce: macce
hu-s.macce.qp: macce/LF/qp
hu-s.utf8: UTF-8/CRLF
lt-s.baltic: baltic
lt-s.cp1257: CP1257
lt-s.ibm775: IBM775
lt-s.iso885913: ISO-8859-13
lt-s.iso88594: ISO-8859-4
lt-s.macce: macce
lt-s.utf8: UTF-8/CRLF
lv-s.baltic: baltic
lv-s.cp1257: CP1257
lv-s.ibm775: IBM775
lv-s.iso885913: ISO-8859-13
lv-s.iso885913.qp: ISO-8859-13/qp
lv-s.iso88594: ISO-8859-4
lv-s.macce: macce
lv-s.utf8: UTF-8/CRLF
pl-s.baltic: baltic/CRLF
pl-s.baltic.qp: baltic/CRLF/qp
pl-s.cork: CORK/LF
pl-s.cp1250: CP1250
pl-s.ibm852: IBM852
pl-s.iso885913: ISO-8859-13
pl-s.iso885916: ISO-8859-16
pl-s.iso88592: ISO-8859-2
pl-s.utf8: UTF-8/CRLF
ru-s.cp1251: CP1251
ru-s.cp866: IBM866
ru-s.iso88595: ISO-8859-5
ru-s.koi8r: KOI8-R
ru-s.maccyr: maccyr
ru-s.maccyr.qp: maccyr/qp
ru-s.utf8: UTF-8/CRLF
sk-s.cork: CORK/LF
sk-s.cp1250: CP1250
sk-s.cp1250.qp: CP1250/qp
sk-s.ibm852: IBM852
sk-s.iso88592: ISO-8859-2
sk-s.keybcs2: KEYBCS2
sk-s.koi8cs2: KOI-8_CS_2
sk-s.macce: macce
sk-s.utf8: UTF-8/CRLF
sl-s.cork: CORK/LF
sl-s.cp1250: CP1250
sl-s.ibm852: IBM852
sl-s.ibm852.qp: IBM852/LF/qp
sl-s.iso88592: ISO-8859-2
sl-s.macce: macce
sl-s.utf8: UTF-8
uk-s.cp1125: CP1125
uk-s.cp1251: CP1251
uk-s.ibm855: IBM855
uk-s.iso88595: ISO-8859-5
uk-s.koi8u: KOI8-U
uk-s.maccyr: maccyr
uk-s.utf8: UTF-8/CRLF
zh-s.big5: BIG5
zh-s.gbk: GBK
zh-s.hz: HZ
zh-s.utf8: UTF-8
none-s.ucs2: UCS-2/LF
none-s.ucs2be.bom: UCS-2/LF
none-s.ucs2be.bom.crlf: UCS-2/CRLF
none-s.ucs2be.bom.noeol: UCS-2/LF
none-s.ucs2le: UCS-2/LF/21
none-s.ucs2le.bom: UCS-2/LF/21
none-s.ucs2le.qp: UCS-2/LF/21/qp
none-s.ucs4: UCS-4/LF
none-s.ucs4le: UCS-4/LF/4321
none-s.utf7: UTF-7/LF
none-s.utf8: UTF-8
none-s.utf8.bom: UTF-8
